ethers-ext/example/browser-react/src/components/KlaytnVT.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import { Account } from '../types';
+import { doSendTx } from '../util';
+import { TxType } from '@klaytn/js-ext-core';
+
+type Props = {
+  account: Account;
+};
+
+function KlaytnVT({ account }: Props) {
+    const [txURL, setTxURL] = useState('');
+
+    async function sendKlaytnVT(address: string) {
```

Comment:
1. Define `handleSubmit` instead of `sendKlaytnVT` to simplify the `<form>` tag
2. In Utils.tsx, modify `doSendTx` to `async function doSendTx(account: Account, txRequest: any): string`
3. In Utils.tsx, add `function getTxhashUrl(chainId: number, txhash: string): string`
4. In App.css, define `.errorMsg` that's red color and bold text.

```tsx
const [txhash, setTxhash] = useState<string>("");
const [error, setError] = useState<any>(null);

async function handleSubmit(e) {
  e.preventDefault();
  const toAddr = getAddress(e.target.to.value);
  const valuePeb = parseKlay(e.target.amount.value);
  const tx = {
    type: TxType.ValueTransfer, // 0x08
    to: toAddr,
    value: valuePeb,
  };

  try {
    const txhash = await doSendTx(account, tx);
    setTxhashUrl(txhashUrl);
  } catch (e: any) {
    setError(e);
  }
}

return (
  <form onSubmit={handleSubmit}>
  </form>
  <a target="_blank" href={getTxhashUrl(txhash)}>tx sent: {txhash}</a>
  <text class="errorMsg">{error}</text>
)
```

Reply:
2.  In Utils.tsx, modify doSendTx to async function doSendTx(account: Account, txRequest: any): string

-> Type StringConstructor is not a valid async function return type in ES5/ES3 because it does not refer to a Promise-compatible constructor value. So I will replace 'string' to 'Promise<any>'

Reply:
@nohkwak Oh, I missed it. Yes, the return type must be `Promise<string>`.

Reply:
4. In App.css, define .errorMsg that's red color and bold text.

-> 'class' keyword not working, so I added the tag <b style={{ color: "red" }}>{error}</b>

Reply:
I updated, thank you.